### PR TITLE
MNT: rerender

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Hi! This is the friendly automated conda-forge-webservice.

I've rerendered the recipe as instructed in #100.

Here's a checklist to do before merging.
- [ ] Bump the build number if needed.

### CODE REVIEW

**Comment by ocefpaf** [comment]
Looks like we can reproduce #101 on the CIs. @xhochy could this be related to the `libssh2` change?

**Comment by isuruf** [comment]
Can you check which dependencies changed from the last successful run?

**Comment by ocefpaf** [comment]
> Can you check which dependencies changed from the last successful run?

```diff
> diff passing.txt failing.txt 
4c4
< cftime:          1.1.1.2-py38h4b8e87e_0    conda-forge
---
> cftime:          1.1.3-py38h40bdaab_0      conda-forge
9c9
< intel-openmp:    2020.0-166                defaults   
---
> intel-openmp:    2020.1-216                defaults   
16,17c16,17
< libnetcdf:       4.7.4-nompi_h256d12c_103  conda-forge
< libssh2:         1.8.2-h642c060_2          conda-forge
---
> libnetcdf:       4.7.4-nompi_h256d12c_104  conda-forge
> libssh2:         1.9.0-h3235a2c_1          conda-forge
24c24
< setuptools:      46.1.3-py38h32f6830_0     conda-forge
---
> setuptools:      46.3.1-py38h32f6830_0     conda-forge
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -70,6 +71,7 @@ requirements:
     - openssh  # [mpi == 'openmpi']
     - mkl 2020.0  # [win]
     - intel-openmp 2020.0  # [win]
+    - libssh2 1.8.*
```
Comment (ocefpaf) [inline recipe/meta.yaml:74]: I'm not sure forcing this here is the better way to fix this. Pretty much everything that depends indirectly on `libssh2` (or `curl`) will be broken due to https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/46, right?

Comment (xhochy) [reply recipe/meta.yaml:74]: I guess this will also be reversed if the repodata are run again.

Comment (ocefpaf) [reply recipe/meta.yaml:74]: Yep.

Comment (isuruf) [reply recipe/meta.yaml:74]: Yes, this was just a test, but the bot merged it

**Comment by dopplershift** [comment]
Is it really necessary to pin to mkl/intel-openmp? By locking to 2020, this became package incompatible with the only windows build of scipy 1.5.0. I'm not saying I have to have netcdf4-1.5.4, but I just lost over an hour trying to figure out why an environment I was trying to set up had conflicts. Needless to say, netcdf4 + scipy as conflicting was not an early guess.

**Comment by isuruf** [comment]
Don't know. Can you send a PR to loosen it?

**Comment by dopplershift** [comment]
Trying in #104 .

File: recipe/meta.yaml (hunk 1 above)
Comment (xylar) [reply recipe/meta.yaml:74]: This is causing trouble and I guess I can take it out now?

**Pull request merged**